src/entry.js

```diff
@@ -1,40 +1,16 @@
-const glob = require('glob');
-const shared = require('./shared');
-
 function StacheEntryPlugin() {
-  let _pluginPaths = glob.sync(
-    './*.js',
-    {
-      ignore: [
-        './*.spec.js',
-        './entry.js',
-        './shared.js'
-      ],
-      cwd: __dirname
-    }
-  );
-
   const preload = (content, resourcePath, skyPagesConfig) => {
-    if (!_pluginPaths.length) {
-      return content;
-    }
-
-    _pluginPaths.forEach(pluginPath => {
-      try {
-        const plugin = require(pluginPath);
-
-        if (!plugin.preload) {
-          return;
-        }
-
-        const altered = plugin.preload(content, resourcePath, skyPagesConfig);
-
-        if (content !== altered) {
-          content = altered;
-        }
-      } catch (error) {
-        throw new shared.StachePluginError(error.message);
```

Comment:
The `throw` from the plugins themselves won't be sufficient to be surfaced as an error elsewhere?